src/geo/cartodb-layer-group-view-base.js

```diff
@@ -57,6 +66,7 @@ CartoDBLayerGroupViewBase.prototype = {
         .on('off', function (o) {
           if (self._interactionDisabled) return;
           o = o || {};
+          // TODO: zera has an .on('error', () => { }) callback that should be used here
```

Comment:
`... that should be used here` -> Why don't we use it then?

Reply:
Since the current issue is generating multiple errors in the track.js and we want to fix it asap IMO we should address the zera thing in a different issue.

Reply:
https://github.com/CartoDB/zera/issues/5